main.go

```diff
@@ -330,15 +402,35 @@ func newReverseProxy(logger *slog.Logger, lc tailscaleLocalClient, url *url.URL)
 			return
 		}
 
-		// Proxy requests from tagged nodes as is.
-		if whois.Node.IsTagged() {
+		// Proxy requests from non-funnel tagged nodes as is.
+		//
+		// TODO(lstoll) figure out why - and if end-user nodes would be tagged?
+		if whois.Node.IsTagged() && !isFunnel {
```

Comment:
yeah I guess a config w/ a list of tags, or maybe node id to consider as end-users??